docs/source/conf.py

```diff
@@ -333,6 +333,7 @@ def get_incollection_template(self, e):
     "examples/bqskit": "_static/bqskit.png",
     "examples/simple-landscape-qiskit": "_static/simple-landscape-qiskit.png",
     "examples/learning-depolarizing-noise": "_static/learn-depolarizing.png",
+    "examples/pec-tutorial": "_static/pec-tutorial.png",
```

Comment:
```suggestion
    "examples/pec_tutorial": "_static/pec-tutorial.png",
```

The thumbnail is not working. Probably this will fix the problem.